src/commands/general/ping.ts

```diff
@@ -1,19 +1,15 @@
-import { useQueue } from 'discord-player';
 import { ChatInputCommandInteraction, SlashCommandBuilder } from 'discord.js';
 export const command = {
     data: new SlashCommandBuilder()
         .setName('ping')
         .setDescription('Get the bots ping')
-        .addBooleanOption(option => option
-            .setName('show')
-            .setDescription('Show to everyone')
-            .setRequired(false)
-        ),
+        .addBooleanOption((option) => option.setName('show').setDescription('Show to everyone').setRequired(false)),
     category: 'General',
-    async execute(interaction : ChatInputCommandInteraction) {
+    async execute(interaction: ChatInputCommandInteraction) {
         let show = interaction.options.getBoolean('show') || false;
```

Comment:
Hey! Can you remove the show variable entirely and just directly set ephemeral to true please. Thanks, nice pull! 😊

Reply:
so this means that it doesn;t show the ping to everyone?

Reply:
I don't really think that showing the ping is useful for anyone other then the person specifically requesting the data.